stock_inventory_hierarchical_location/__openerp__.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    This module is copyright (C) 2013 Numérigraphe SARL. All Rights Reserved.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    "name": "Exhaustive and hierarchical Stock Inventories",
```

Comment:
In `__openerp__.py` you can use the dict key `summary` as in https://github.com/OCA/vertical-ngo/blob/8.0/logistic_order_donation/__openerp__.py

It is just a short description which is shown on apps and in module list which helps the user to figure out what does the module. Many OCA modules are missing it but I think we should use it as it is quite useful.

Here you see some modules with a summary bellow the title (though many are missing it)
https://www.odoo.com/apps/modules?series=8.0&order=Downloads&author=Odoo+Community+Association+%28OCA%29